x-pack/solutions/observability/plugins/observability/common/constants.ts

```diff
@@ -35,6 +35,15 @@ export const observabilityRuleCreationValidConsumers: RuleCreationValidConsumer[
 
 export const EventsAsUnit = 'events';
 
+export const LEGACY_UPTIME_MONITOR_STATUS_RULE_ID = 'xpack.uptime.alerts.monitorStatus';
+export const LEGACY_UPTIME_TLS_RULE_ID = 'xpack.uptime.alerts.tlsCertificate';
+export const LEGACY_UPTIME_DURATION_ANOMALY_RULE_ID = 'xpack.uptime.alerts.durationAnomaly';
+export const LEGACY_UPTIME_RULE_IDS = [
```

Comment:
Can we use this list instead of defining these constants here? We centralized rule IDs in `@kbn/rule-data-utils` package to not define it in every plugin separately.

https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-rule-data-utils/src/rule_types/o11y_rules.ts#L61

```
export const UPTIME_RULE_TYPE_IDS = [
  'xpack.uptime.alerts.tls',
  'xpack.uptime.alerts.tlsCertificate',
  'xpack.uptime.alerts.monitorStatus',
  'xpack.uptime.alerts.durationAnomaly',
];
```

Shouldn't we also have `xpack.uptime.alerts.tls` here? (not sure what us the difference between tls and tlsCertificate)

Reply:
++ I was hoping something like this existed but wasn't sure where it would be.

Reply:
See @shahzad31's review below, we're likely going to modify this fix to work differently, but we can still use this.